Foundation/FileManager.swift

```diff
@@ -2377,46 +2377,113 @@ extension FileManager {
             self.innerEnumerator = ie
         }
         
-        @available(Windows, deprecated, message: "Not Yet Implemented")
         override func nextObject() -> Any? {
             let o = innerEnumerator.nextObject()
             guard let url = o as? URL else {
                 return nil
             }
 
 #if os(Windows)
-            NSUnimplemented()
+            let path = url.path.replacingOccurrences(of: baseURL.path+"\\", with: "")
 #else
             let path = url.path.replacingOccurrences(of: baseURL.path+"/", with: "")
+#endif
             _currentItemPath = path
             return path
-#endif
         }
     }
 
-#if os(Windows)
     internal class NSURLDirectoryEnumerator : DirectoryEnumerator {
-        internal typealias ErrorHandler = /* @escaping */ (URL, Error) -> Bool
+#if os(Windows)
+        var _url : URL
```

Comment:
Is `_url` needed? It doesnt look to be used outside of `init()` since it is also stored in `_stack`.

Reply:
Nope, removed.